lib/metrics/prometheus.js

```diff
@@ -47,49 +52,49 @@ class PrometheusMetric {
     }
 
     increment(amount, labels) {
-        this._handleStaticLabels(labels);
-        labels = labels.map(this._normalize);
-        this.metric.labels.apply(this.metric, labels).inc(amount);
+        let updatedLabels = this._handleStaticLabels(labels);
```

Comment:
const. it can be const now.

Reply:
But we are re-assigning it in the next line

Reply:
Can you merge the lines? 

```
const updatedLabels = this._handleStaticLabels(labels).map(this._normalize);
```
?

Reply:
const is always much better then let.